fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -3262,7 +3253,7 @@ public void testBinarySetBadMimeType() throws IOException {
                 "INSERT { <" + subjectURI + "> ebucore:hasMimeType \"-- invalid syntax! --\" } WHERE {}")
         );
 
-        assertEquals(BAD_REQUEST.getStatusCode(), getStatus(patch));
```

Comment:
Are you saying that a malformed SPARQL-Update request should silently fail? It seems like `BAD_REQUEST` is a more correct response. Am I missing something?

Reply:
Its actually not invalid syntax unless we want to parse all ontologies for their targets and domains.

This was a problem because we relied on this particular triple to send the binary response. Now we store the mime-type in the headers file and this is user rdf. If they want the triple `<> ebucore:hasMimeType "I want my baby back baby back baby back"` that is up to them. 

I could modify this test to check that the binary is still retrievable and the user rdf says whatever or I can remove the entire test.